alfresco-zipkin-repo/build.gradle

```diff
@@ -1,6 +1,8 @@
 plugins {
     id 'eu.xenit.alfresco' version "0.2.0"
     id "eu.xenit.amp" version "0.2.0"
+    id "idea"
```

Comment:
Don't apply the `idea` plugin to your projects

Reply:
done in latest commit